R/add_md.R

```diff
@@ -55,7 +55,7 @@ shove_nodes_in <- function(body, new, nodes, where = "after", space = TRUE) {
     xpath <- NULL
   }
   if (length(nodes) == 0) {
-    msg <- glue::glue("No nodes matched the expression {sQuote(xpath)}")
+    msg <-  cli_text("No nodes matched the expression {sQuote(xpath)}")
```

Comment:
There are two things here that can be addressed:  

1. Adding `cli::` to the beginning of the call lets R know where to fetch the function from (we _could_ import this from the {cli} package, but we've made the choice not to because it makes it more clear where the function is coming from).
2. using `cli::format_error()` instead of `cli::cli_text()`. The `cli::format_` functions are designed specifically for `rlang::abort()` and similar: https://cli.r-lib.org/reference/format_error.html


```suggestion
    msg <-  cli::format_error("No nodes matched the expression {sQuote(xpath)}")
```